scraper/src/mindtouch2zim/entrypoint.py

```diff
@@ -159,29 +135,28 @@ def main(tmpdir: str) -> None:
         "--overwrite",
         help="Do not fail if ZIM already exists, overwrite it",
         action="store_true",
+        dest="overwrite_existing_zim",
     )
 
-    # ZIM configuration flags
-    add_zim_config_flags(parser, zim_defaults())
-
-    # Document selection flags
-    add_content_filter_flags(parser)
-
     parser.add_argument(
         "--output",
         help="Output folder for ZIMs. Default: /output",
+        type="Path",
```

Comment:
Why as a string?

```suggestion
        type=Path,
```

Reply:
A registered a type, but this is indeed completely overkill